images/ig-publisher-base/README.md

```diff
@@ -92,3 +92,29 @@ resources in your host filesystem, you can mount them from elsewhere. For
 instance, to bring in the `.fhir` cache from your host machine home directory:
 
     -v /home/$USER/.fhir:/home/publisher/.fhir
+
+## Running on WSL2
+Using this image to build your IG using Docker on WSL2 is really slow due to a 
+known limitation when mounting the Windows filesystem. This can be circumvented
```

Comment:
```suggestion
known limitation when mounting the Windows filesystem. You can run the IG publisher without docker by following the instructions at https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation#IGPublisherDocumentation-Installing; or you can improve performance in WSL2
```

Reply:
Added a proposed edit to the PR, to address Grahame's point; see if you're okay with it @LodewijkSioen ?

Reply:
Okay for me. But I think that people looking at this docker image will already know this. We want to use docker as a way to minimize the number of dependencies people have to install on their PC when working on IGs.